### ISSUE TITLE
Social Feed in colums

### ISSUE THREAD

**Issue by papazetis** [pavelk2/social-feed#273]
Hi.

Can I use social feed in columns?
I need three columns that the 1st is for Twitter, the 2nd for FB and the 3rd for Instagram.

I want in each column to show only 2 posts for each social network.

Is it possible to do it?

**Comment by HartLarsson** [comment]
No, because the script collect all feeds in one stream

**Comment by mutiullah7** [comment]
@papazetis you can make it masonry like stream using css the column-count property like
```
.social-feed-container {
    column-gap: 0 !important;
    column-count: 4;
    -webkit-column-count: 4;
    -moz-column-count: 4;
}
```
it will credate 4 column feed

**Comment by HartLarsson** [comment]
yes he could but he cannot split in columns "one column-per-social" as he asked

**Comment by mutiullah7** [comment]
mmm hmmm, you are right @HartLarsson
